CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#152](https://github.com/nrepl/nrepl/issues/60): Bug fix for 'close' operation to kill session thread
```

Comment:
Seems you forgot the update that link to point to the actual issue. Bug fixes should go under "Bugs fixed", not under new features.